readme.md

```diff
@@ -5,14 +5,23 @@
 =====
 Automatic Video Library Manager for TV Shows. It watches for new episodes of your favorite shows, and when they are posted it does its magic.
 
-#### Features
+#### Exclusive features
+ - Supports TVMaze and TMDB indexers
  - Manual search episodes (you choose what to snatch based on all kind of info: provider, seeds, release group)
  - Manual search for subtitles (useful when subtitle has low score because of typo in filename or alternative title)
  - Uses 'guessit' library to parse and enrich existing info (more precise than regexes)
  - Improved Anime shows support
+ - Faster DailySearcher as we only process new items. Already processed items are discarded.
+ - Option to clean Kodi library when replacing existing files
+ - Better quality explanations and consistent quality code in all threads
+ - See on the fly if your backlogged episodes will be increased/reduced while changing qualities
+ - Remove torrent from Transmission (alpha feature) when ratio reached
```

Comment:
Maybe add, Nightly showupdater updates only the season of the show, which has been updated by the indexer, for tvdb and tmdb.